draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -373,10 +380,12 @@ ignored.  If a broken catalog zone was transferred, the newly transferred
 catalog zone MUST be ignored (but the older copy of the catalog zone SHOULD
 be left running subject to values in SOA fields).
```

Comment:
I'm in favor of removing this sentence.
```
If a broken catalog zone was transferred, the newly transferred
catalog zone MUST be ignored (but the older copy of the catalog zone SHOULD
be left running subject to values in SOA fields).
```

Reply:
Only processing the zone when it arrives will solve the expiry issues.
• The content of a catalog zone is processed only once.
• After the processing has finished the zone is no longer special in any way

Reply:
+1

Reply:
removed